src/main/java/ru/scamburger/Soundger/entity/AuthToken.java

```diff
@@ -0,0 +1,22 @@
+package ru.scamburger.Soundger.entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Data
+public class AuthToken {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long user_id;
+
+    @OneToOne(mappedBy="authToken", cascade=CascadeType.ALL)
```

Comment:
Судя по инфе с инета, при удалении строчки AuthToken из бд CascadeType.ALL повлечет за собой удаление связанной сущности (в нашем случае - User).

> It seems in your case to be a bad idea, as removing an XXX would lead to removing the related YYY. 

https://stackoverflow.com/a/13027444

> Есть несколько типов каскадных операций, мы рассмотрим три основные: PERSIST, MERGE и REMOVE. CascadeType.ALL означает, что необходимо выполнять каскадно сразу все операции:

https://sysout.ru/tipy-cascade-primer-na-hibernate-i-spring-boot/

Кажется нам нужно использовать DETACH, но лучше вам разобраться с доками самостоятельно:
![image](https://user-images.githubusercontent.com/51204805/180663815-2144e82c-ea63-4a38-8cc8-cfce6efc34de.png)
https://vladmihalcea.com/a-beginners-guide-to-jpa-and-hibernate-cascade-types/

Смысл в том, что нам нужен такой каскейд, который при удалении самого authToken по сути ничего с юзером сделать не должен.

Reply:
Удаляется только authToken, user'a не трогает
Но в обратном случае если мы удаляем user'a то и authToken стирается

Reply:
тогда окей